README.md

```diff
@@ -71,6 +71,46 @@ testAgent.on('metrics', console.log)
 testAgent.on('process.memory', console.log)
 ```
 
+## Metric Tags
+
+The agent is able to use environment variables as metrics tags. 
+Define a list of environment variables to be used as metric tags:  
+
+```
+# add the values of env. variables USER, PWD as tags to metrics
+export MONITORING_TAGS_FROM_ENV="USER, PWD"
```

Comment:
Don't USER and PWD need to be $USER and $PWD?

Reply:
1) In bash, yes. But should it be like that in our config? This could cause trouble. 
2) The variable name is USER. Try `env` in your console. Or when you set a variable you write `export USER=me`. In JS we access env variables via `process.env.USER`.  3) Potential problems: Imagine what would happen when you would use $USER in the above statement. Bash would replace $USER with `me` and this is not what we want - imagine you pass the config to a container, then we have no shell evaluating $USER variable. We want to tell the agent to collect the value from process.env.USER at runtime in the process environment.
Conclusion: No changes.

Reply:
$ export MONITORING_TAGS_FROM_ENV="USER, PWD"
$ echo $MONITORING_TAGS_FROM_ENV 
USER, PWD

Probably not what the person wants or thinks they will get if they copy-paste this example.

Not sure this is what you/they want either:
$ export MONITORING_TAGS_FROM_ENV="$USER, $PWD"
$ echo $MONITORING_TAGS_FROM_ENV 
otis, /Users/otis/Downloads